search.py

```diff
@@ -1,67 +1,73 @@
-import requests
 import click
-import webbrowser
 import pandas
-import os
+import time
 from bs4 import BeautifulSoup
+from bs4.element import Tag
 from pathlib import Path
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 
 
 @click.command()
 @click.option('-s', '--search', "user_query", default='World', help="User's search query")
-@click.option('-b', '--browser', "browser", default='chrome', help="Browser to use")
+@click.option('-b', '--browser', "browser", default='Chrome', help="Browser to use")
 @click.option('-o', '--output', "output", default='not', help="File to save the results")
 def main(user_query, browser, output):
     print(f"Searching for {user_query} in {browser}...")
+    if browser.capitalize() == 'Firefox':
```

Comment:
extract logic of the opening browser to other class

Reply:
Done.